include/klee/Expr.h

```diff
@@ -1080,7 +1080,8 @@ class ConstantExpr : public Expr {
   }
 
   static ref<ConstantExpr> create(uint64_t v, Width w) {
-    assert(v == bits64::truncateToNBits(v, w) && "invalid constant");
+    if (w <= 64)
```

Comment:
In my version I surrounded the check with `#ifndef NDEBUG` because the assert does nothing in a non assert build.